synapse/config/password_auth_providers.py

```diff
@@ -52,7 +52,18 @@ def read_config(self, config, **kwargs):
 
     def generate_config_section(self, **kwargs):
         return """\
-        #password_providers:
+        # Password providers allow homeserver administrators to integrate
+        # their Synapse installation with existing authentication methods
+        # ex. LDAP, external tokens, etc.
+        #
+        # For more information and known implementations, please see
+        # https://github.com/matrix-org/synapse/blob/master/docs/password_auth_providers.md
+        #
+        # Note: instances wishing to use SAML or CAS authentication should
+        # use the `saml2_config` option instead.
```

Comment:
Is mentioning CAS right here?

Reply:
Probably, the configuration for that is `cas_config`.

Reply:
Ah ok, thanks!